tests/python/test_multidevice.py

```diff
@@ -1688,3 +1688,78 @@ def test_transformer_backward(multidevice_test, benchmark):
     _assert_shape_dtype(inp_grad, [b, s, e], torch.bfloat16)
 
     benchmark.pedantic(benchmark_fn, rounds=5)
+
+
+class OverlapAGMatmulStreamOutermost(FusionDefinition):
```

Comment:
I'd create a separate file for this, e.g., test_multidevice_executor. I should have started doing that for other tests in this file -- I can see it soon become another test_gpu1.cpp :)

Reply:
done at `tests/python/test_multidevice_overlap.py`